src/components/Search/Filter.js

```diff
@@ -213,8 +254,21 @@ class FilterForm extends React.Component {
 };
 
 const Filter = withFormik({
-	 async handleSubmit(values, { props, setSubmitting }){
-		const {
+    mapPropsToValues() {
+        return ({
+            category : 'buy',
+			type : 'any',
```

Comment:
type: {value: "any", label: "Any"} , this thing is not working by the way, mostly because your form element is not a controlled component, it's ok for now just change the type feild